src/mocks/handlers.ts

```
@@ -0,0 +1,361 @@
+import { http, HttpResponse } from 'msw';
+import { mockDb, ensureOrgCollections, randomId } from './mockData';
+
+const ALLOWED_ROLES = new Set(['owner', 'admin', 'attorney', 'paralegal'] as const);
+type Role = 'owner' | 'admin' | 'attorney' | 'paralegal';
+
+function isValidRole(role: unknown): role is Role {
+  return typeof role === 'string' && ALLOWED_ROLES.has(role as Role);
+}
+import type { MockPractice, MockInvitation } from './mockData';
+
+type Role = 'owner' | 'admin' | 'attorney' | 'paralegal';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Role'. Consider to delete it or rename it.

'Role' is defined here:


(lint/suspicious/noRedeclare)